src/modules/addRemove.js

```diff
@@ -0,0 +1,172 @@
+/* eslint-disable max-classes-per-file */
+import Task from './task.js';
+
+class Store {
+  static getTasks() {
+    let tasks;
+    if (localStorage.getItem('tasks') === null) {
+      tasks = [];
+    } else {
+      tasks = JSON.parse(localStorage.getItem('tasks'));
+    }
+
+    return tasks;
+  }
+
+  static addTask(task) {
+    const tasks = Store.getTasks();
+    tasks.push(task);
+    localStorage.setItem('tasks', JSON.stringify(tasks));
+  }
+
+  static removeTask(elem) {
+    if (elem.classList.contains('remove-btn')) {
+      const tasks = Store.getTasks();
+      if (tasks.length === 1) {
+        tasks.pop();
+      } else {
+        tasks.splice(parseInt(elem.id, 2), 1);
+
+        tasks.forEach((element, index) => {
+          element.id = index + 1;
+        });
+      }
+      index = tasks.length;
+
+      localStorage.setItem('tasks', JSON.stringify(tasks));
+    }
+  }
+
+  static modifyTask(elem) {
```

Comment:
- [ ] After modifying the task, when I refresh the page, the task reset to the previous `task name`.It means the modifying task didn't get saved into `local storage`. Kindly fix this

Reply:
@Deepakdanger Everything is ok now, I fix the issue.